tools/SS13SmoothingCutter/Cutter.dm

```diff
@@ -218,9 +218,9 @@
 				diagnw1.Turn(180)
 				outputIcon.Insert(diagnw1, "d-nw-0")
 
-				to_world("Diag_Corner_B: \icon[diag_corner_b] -> \icon[diagse1] \icon[diagsw1] \icon[diagne1] \icon[diagnw1]")
+				world << "Diag_Corner_B: \icon[diag_corner_b] -> \icon[diagse1] \icon[diagsw1] \icon[diagne1] \icon[diagnw1]"
 
 
 
 		fcopy(outputIcon, filename)	//Update output icon each iteration
-	to_world("Finished [filename]!")
+	world << "Finished [filename]!"
```

Comment:
```suggestion
	to_world("Finished [filename]!")
```

Reply:
Arrow, those changes were made by putting in a functioning copy of SS13smoothingCutter to replace the butchered one that is currently in master.  The proc to_world does not exist in the cutter program, and what you are essentially doing is asking me to re-break it.

That seems counter-productive to me, since I just fixed it. With to_world in those spots, it won't function.  Now, it does.

Reply:
So remake it?

Reply:
wat

Reply:
... You want me to rewrite a program that used outside of the server, and is designed to cut up images to be used in the smoothing function solely for the purpose of making to_world work when it works perfectly fine as written here.

That is out of the scope of this PR, and utterly ridiculous.

Reply:
Hm actually yeah. Thats a external tool. So its fine